MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2750,6 +2750,8 @@ public string GetProgressiveVideoArguments(EncodingJobInfo state, EncodingOption
                 args += " -mpegts_m2ts_mode 1";
             }
 
+            var supportsGlobalHeaderFlag = state.OutputContainer != "mkv";
```

Comment:
shouldn't this be changed to something like
```cs
var globalHeaderRequired = state.OutputContainer == "mkv";
```

and then use it like
```cs
if (!globalHeaderRequired)
{
    args += " -flags -global_header";
}
```
?

Reply:
Yes but I was waiting on what the outcome of this discussion is. For me it did not seem final.

Reply:
I think we should rely on ffmpeg being smart and tune as little as possible, but let's start small and turn the flag off in the case where it is really causing us pain (we can turn it off for other cases later).

Reply:
Well all our outputs require the flag to be there. I'm not sure why we removed it anyway. Both DASH, HLS and segment require it. So what is this flag doing here anyway.

Reply:
_because poetry, man!.._

Reply:
Do you mean just removing appending `"-flags -global_header"` and let ffmpeg decide?

Reply:
This would be ideal if it would solve all our problems, but it makes testing your change harder. However, we can be bold and try things! Let's go and remove this flag everywhere, maybe nothing would break... 🤞

Reply:
I have removed the `global_header` flag and tested it on my setup and as expected it worked on my phone. I'll mark this as resolved until further notice.